change-validation/src/main/scala/com/normation/plugins/changevalidation/ValidationNeeded.scala

```diff
@@ -191,7 +191,7 @@ class NodeGroupValidationNeeded(
     val directiveId = change.newDirective.id
     val start = System.currentTimeMillis()
     val res = for {
-      rules     <- ruleLib.getAll(includeSytem = true).map( _.filter(r => r.directiveIds.contains(directiveId))).toBox
+      rules     <- ruleLib.getAll(includeSystem = true).map( _.filter(r => r.directiveIds.contains(directiveId))).toBox
```

Comment:
@Fdall THis would break the plugin as this is part of Rudder's APIs. I already started the typos check on plugins but it must be done manually, if possible by a scala dev for scala plugins to avoid breaking stuff.

Reply:
I will add a temporary exception in the repo